CHANGELOG.md

```diff
@@ -193,6 +193,7 @@ FuelVM received a lot of safety and stability improvements:
 - Refactored errors, returning more detailed errors to the user, simplifying debugging.
 
 ### Added
+- [#1519](https://github.com/FuelLabs/fuel-core/pull/1519): Stream chain state from file
```

Comment:
The breaking changes can be added in the final feature branch to better summarise them